repository.yml

```diff
@@ -127,3 +127,11 @@ repo.deps:
             master: 0-dev
             mynewt_1_8_0_tag: 0.1.0
             mynewt_1_9_0_tag: 0.2.0
+
+    mynewt-memfault-sdk:
+        type: github
```

Comment:
Fix indentation here to be consistent.

Reply:
I'm unable to see the difference in indent here.

Reply:
Actually, you are right. It is fine. All good.